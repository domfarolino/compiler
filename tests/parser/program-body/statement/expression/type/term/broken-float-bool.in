program dominic is

  integer result;

begin

  result := 132.242 * true;

end program.
