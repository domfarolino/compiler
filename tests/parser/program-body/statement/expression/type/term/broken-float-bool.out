[1;31mLine 7: [Term]: float and bool cannot be multiplied or divided
Line 7: Error parsing program body
[0m